stats/DataAggregation.md

```diff
@@ -0,0 +1,67 @@
+# Data Aggregation API Overview
+The stats library allows users to define measurements how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down by tags and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library SHOULD provide support for multiple types of Aggregations:
+* `Count`: indicates that data collected and aggregated with this `Aggregation` type will be turned
+into a count value.
```

Comment:
Something like "Count: counts the number of measurements recorded" might be clearer.

Reply:
done.